src/util/side_metadata/constants.rs

```diff
@@ -22,6 +22,8 @@ pub(crate) const LOG_GLOBAL_SIDE_METADATA_WORST_CASE_RATIO: usize = 1;
 /// So, a value of `n` means this ratio must be less than $2^-n$.
 #[cfg(target_pointer_width = "32")]
 pub const LOG_LOCAL_SIDE_METADATA_WORST_CASE_RATIO: usize = 3;
+#[cfg(target_pointer_width = "64")]
+pub const LOG_LOCAL_SIDE_METADATA_WORST_CASE_RATIO: usize = 1;
```

Comment:
The reason we didn't have it was that it was not used anywhere.
In fact, the only place it could have been used was in `mallocspace`'s metadata, to calculate the offset of `MARKING_METADATA_SPEC` in the 64-bits case.
Instead, the implementation uses the actual size of `ALLOC_METADATA_SPEC` instead of its maximum size (which depends on `LOG_LOCAL_SIDE_METADATA_WORST_CASE_RATIO `), to calculate the offset.

With the sanity checker, we need this constant to check whether the local metadata size is reasonable or not.